### PR TITLE
@/climatemappedafrica Page Hero

### LINKED ISSUE #920: ClimateMapped About Page Header Block
Create ClimateMapped About Page Header Block

Request Link: [Feature 2](https://docs.google.com/document/d/1TUrdCRnbj6TSqiuwITKUpnDUEi_ktm_3PmgsaD0y2bw/edit#heading=h.7lvx2h28muyq)

> Preview
Image

> Block fields
Image

### PR DESCRIPTION
**Pull request by kilemensi**
## Description

This PR implements the Page Hero for @/climatemappedafrica app's /about.

Fixes #920.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots

S

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: apps/climatemappedafrica/src/pages/[[...slug]].js [hunk 1]
```diff
@@ -3,10 +3,12 @@ import { SWRConfig } from "swr";
```
Comment (koechkevin) [inline apps/climatemappedafrica/src/pages/[[...slug]].js:3]: Should we name this file `[[...slugs]].js` ?

**Review by koechkevin — APPROVED** [review]

File: apps/climatemappedafrica/src/pages/[[...slug]].js (hunk 1 above)
Comment (kilemensi) [reply apps/climatemappedafrica/src/pages/[[...slug]].js:3]: Not sure I follow Mr. @koechkevin; isn't it named `[[...slug]].js` already?

Comment (koechkevin) [reply apps/climatemappedafrica/src/pages/[[...slug]].js:3]: I'm just asking @kilemensi we've named slugs in other apps(charterafrica, codeforafrica etc) so I thought we should do the same as well here.

Comment (kilemensi) [reply apps/climatemappedafrica/src/pages/[[...slug]].js:3]: Oh `slugs` vs `slug`! I didn't even see that... I didn't create this file so I'm OK with what you guys think is best.

Next.js seem to be using `slug` [though](https://nextjs.org/docs/pages/building-your-application/routing/dynamic-routes#optional-catch-all-segments).

**Comment by koechkevin** [comment]
@kilemensi Shouldn't this PR go 🚀?

**Comment by kilemensi** [comment]
>@kilemensi Shouldn't this PR go 🚀?

Daamn, thought it was already merged @koechkevin ...

**Pull request merged**